### PR TITLE
fix(chat): fix slider overload and keyboard arrows navigation (Issue #2825)

### PR DESCRIPTION
**Pull request by Alexander-Kezik**
**Description:**

fix slider overload and keyboard arrows navigation

Issues:

- Issue #2825

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2862.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Chat/TalkTo/TalkToCard.tsx [hunk 1]
```diff
@@ -225,7 +225,7 @@ export const TalkToCard = ({
         Icon: UnpublishIcon,
         onClick: (e: React.MouseEvent) => {
           e.stopPropagation();
-          onPublish?.(entity, PublishActions.DELETE);
+          onPublish(entity);
```
Comment (IlyaBondar) [inline apps/chat/src/components/Chat/TalkTo/TalkToCard.tsx]: Why does unpublish menu option execute `onPublish` (ADD action as result)?

Comment (Alexander-Kezik) [reply apps/chat/src/components/Chat/TalkTo/TalkToCard.tsx:228]: because it's publish request anyway

Comment (Alexander-Kezik) [reply apps/chat/src/components/Chat/TalkTo/TalkToCard.tsx:228]: Because we agreed on hide unpublish option from talk to cards

Comment (Alexander-Kezik) [reply apps/chat/src/components/Chat/TalkTo/TalkToCard.tsx]: option removed

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2862.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**